src/main/java/org/parabot/api/misc/Version.java

```diff
@@ -8,7 +8,8 @@ public Version(String version) {
         if (version == null) {
             throw new IllegalArgumentException("Version can not be null");
         }
-        if (!version.matches("[0-9]+(\\.[0-9]+)*")) {
+
+        if (!version.matches("[0-9]+(\\.[0-9]+)*") && !version.matches("[0-9]+(\\.[0-9]+)*-RC-([0-9]+)")) {
```

Comment:
Could you explain this logic?

Reply:
For the launcher to support nightly builds this is needed, as it initializes an instance of Version here: https://github.com/Parabot/Launcher/blob/3b0028af9da0376a19dabb937026928b1f18ad34/src/main/java/org/parabot/launcher/helpers/Launcher.java#L36
But since the nightly builds have a different version number build up it doesn't pass the regular expression in the Version constructor, hence the added regular expression to allow this.